### PR TITLE
Improve the refdocs

### PR DESCRIPTION
**Pull request by saragluna**
Fix some issues found from the reference docs.

### CODE REVIEW

File: docs/src/main/asciidoc/spring-cloud-stream-support.adoc [hunk 1]
```diff
@@ -285,7 +285,7 @@ spring:
         client-id: ${SERVICE_PRINCIPAL_ID}
```
Comment (yiliuTo) [inline docs/src/main/asciidoc/spring-cloud-stream-support.adoc:285]: Please help to update "${SERVICE_PRINCIPAL_ID}" to "${AZURE_CLIENT_ID}" thanks, same for "${SERVICE_PRINCIPAL_SECRET}"

File: docs/src/main/asciidoc/spring-data-support.adoc [hunk 2]
```diff
@@ -264,5 +263,5 @@ Autowired UserRepository interface, then can do save, delete and find operations
 Please refer to link:https://github.com/Azure-Samples/azure-spring-boot-samples/tree/spring-cloud-azure_{project-version}/cosmos[azure-spring-boot-samples] for more details.
 
 
-Besides, using this Azure Cosmos DB Spring Boot Starter, you can directly use Spring Data for Azure Cosmos DB package for more complex scenarios. Please refer to link:https://github.com/Azure/azure-sdk-for-java/tree/spring-cloud-azure-dependencies_{project-version}/sdk/cosmos/azure-spring-data-cosmos[Spring Data for Azure Cosmos DB] for more details.
+Apart from using the `spring-cloud-azure-starter-data-cosmos` library you can directly use `azure-spring-data-cosmos` library for more complex scenarios. Please refer to link:https://github.com/Azure/azure-sdk-for-java/tree/spring-cloud-azure-dependencies_{project-version}/sdk/cosmos/azure-spring-data-cosmos[Spring Data for Azure Cosmos DB] for more details.
```
Comment (Netyyyy) [inline docs/src/main/asciidoc/spring-data-support.adoc:266]: add “，” after library

File: docs/src/main/asciidoc/configuration.adoc [hunk 3]
```diff
@@ -51,37 +51,46 @@ There are some properties that could be shared among different Azure services, f
 
 NOTE: Properties configured under each Azure service will override the global configurations.
 
-Spring Cloud Azure unifies configuration properties' prefixes to `spring.cloud.azure` since 4.0, which will make configuration properties more consistent and more intuitive. Here's a quick review of the prefixes of supported Azure services.
+Spring Cloud Azure unifies configuration properties' prefixes to `spring.cloud.azure` since 4.0, which will make configuration properties more consistent and more intuitive. Here's a quick review of the serivce specific properties.
 
 .Service specific properties
-[cols="<50,<50", options="header"]
+[cols="<3,<3,<3", options="header"]
```
Comment (yiliuTo) [inline docs/src/main/asciidoc/configuration.adoc:57]: I am thinking about if we can add JMS related properties in this table or not, but given those are not prefixed with `spring.cloud.azure`, so perhaps we can introduce the jms property link with another paragraph after the table?

File: docs/src/main/asciidoc/secret-management.adoc [hunk 4]
```diff
@@ -112,9 +112,10 @@ If key exists in multiple PropertySources, which will take effect is decided by
 |===
 
 - Please refer to link:index.html#authorize-access-with-azure-active-directory[Authorize access with Azure AD] to make sure the link:https://docs.microsoft.com/azure/active-directory/develop/app-objects-and-service-principals#service-principal-object[security principal] has been granted the sufficient permission to access the Azure Key Vault Secrets.
-- If common properties like `client`, `credential`, `profile`, `proxy`, `retry` are not configured in `spring.cloud.azure.keyvault.secret.property-sources[].xxx`, `spring.cloud.azure.xxx` will be used. Please refer to link:index.html#configuration[Configuration] to get more information about these common properties.
+- If common properties like `client`, `credential`, `profile`, `proxy`, `retry` aren't configured in `spring.cloud.azure.keyvault.secret.property-sources[].xxx`, `spring.cloud.azure.xxx` will be used. Please refer to link:index.html#configuration[Configuration] to get more information about these common properties.
```
Comment (backwind1233) [inline docs/src/main/asciidoc/secret-management.adoc:115]: line:104 is missed
image

File: docs/src/main/asciidoc/configuration.adoc (hunk 3 above)
Comment (saragluna) [reply docs/src/main/asciidoc/configuration.adoc:57]: this is for the level one sdk configuration, so the jms won't be included in this table. That's why we have a whole section for JMS.

**Review by yiliuTo — APPROVED** [review]
LGTM

**Pull request merged**